### PR TITLE
Add core weekly 4

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2021-02-01-coreweekly-04-2021.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 25th to Sunday 31th of January 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## Releases
+
+* [slint-config](https://github.com/PrestaShop/eslint-config): [v0.1.0](https://github.com/PrestaShop/eslint-config/releases/tag/0.1.0)
```
Comment (ttoine) [inline news/_posts/core-weekly/2021-02-01-coreweekly-04-2021.md]: does this need an alias, as it has been done for modules and other tools?

Comment (Progi1984) [inline news/_posts/core-weekly/2021-02-01-coreweekly-04-2021.md]:
```suggestion
* [eslint-config](https://github.com/PrestaShop/eslint-config): [v0.1.0](https://github.com/PrestaShop/eslint-config/releases/tag/0.1.0)
```

Comment (matks) [reply news/_posts/core-weekly/2021-02-01-coreweekly-04-2021.md]: Just a mistake. I wanted to remove the lowercase e and replace it with an uppercase E but I did only half of the work. Fixed.

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**